WireAnalytics/Package@swift-5.10.swift

```diff
@@ -10,26 +10,33 @@ let package = Package(
         .library(name: "WireAnalytics", targets: ["WireAnalytics"]),
         .library(name: "WireDatadog", targets: ["WireDatadog"])
     ],
-    dependencies: [
-        .package(url: "https://github.com/DataDog/dd-sdk-ios.git", exact: "2.16.0")
-    ],
     targets: [
         .target(
             name: "WireAnalytics",
-            dependencies: resolveWireAnalyticsDependencies(),
-            swiftSettings: swiftSettings
+            dependencies: resolveWireAnalyticsDependencies()
         ),
         .target(
             name: "WireDatadog",
             dependencies: [
-                .product(name: "DatadogCore", package: "dd-sdk-ios"),
-                .product(name: "DatadogCrashReporting", package: "dd-sdk-ios"),
-                .product(name: "DatadogLogs", package: "dd-sdk-ios"),
-                .product(name: "DatadogRUM", package: "dd-sdk-ios"),
-                .product(name: "DatadogTrace", package: "dd-sdk-ios")
-            ],
-            swiftSettings: swiftSettings
+                .target(name: "Datadog")
+//                .product(name: "DatadogCore", package: "Datadog"),
+//                .product(name: "DatadogCrashReporting", package: "dd-sdk-ios"),
+//                .product(name: "DatadogLogs", package: "dd-sdk-ios"),
+//                .product(name: "DatadogRUM", package: "dd-sdk-ios"),
+//                .product(name: "DatadogTrace", package: "dd-sdk-ios")
+            ]
+        ),
```

Comment:
with this target, I have an error when I import DatadogCore or other as it does not find it, I am not sure how to declare these modules here